internal/action/command.go

```diff
@@ -983,6 +983,7 @@ func (h *BufPane) ReplaceCmd(args []string) {
 				// skip empty match right after previous match
 				if searchLoc == end {
 					searchLoc = start
+					prevMatchEnd = buffer.Loc{-1, -1}
```

Comment:
It's better to squash this fix into the first commit.

Reply:
done